zogy_main.tex

```diff
@@ -634,6 +634,123 @@ \section{The decorrelation afterburner\label{sec:decorrab}}
+            \frac{\frac{\overline{\hat{P}_n}}{ F_n} }
+            % denom
+            { \frac{\sigma_n^2}{F_n^2} +
+            \frac{\sigma_r^2}{F_r^2}
+            \frac{\abs{\hat{P}_{mk}}^2}{\abs{\hat{P}_{n}}^2}
+            } \hat{N}
+            -
+            \frac{
+            \frac{\abs{\hat{P}_n}^2}{\hat{P}_rF_r} }
+            % denom
+            { \frac{\sigma_n^2}{F_n^2} +
+            \frac{\sigma_r^2}{F_r^2}
+            \frac{\abs{\hat{P}_{mk}}^2}{\abs{\hat{P}_{n}}^2}
+            } \hat{R}\label{eq:Scncr}
+\end{align}
+%
+\par In \Cref{eq:SPre}, the first term in the numerator is the PSF
+pre-convolved science image that now has a PSF of
+\(\abs{\hat{P}_n}^2\), and the theoretical PSF matching kernel is
+written in \Cref{eq:Smk}. We use the matching kernel to express the
+correction in \Cref{eq:SPre} on the right side. Compared to the
+original decorrelation afterburner expression, the correction is
+squared and the correction with the matching kernel is deconvolved
+with the pre-convolution kernel.
+%
+\par Let's consider the convergence properties towards high
+frequencies in case of Gaussian PSFs. The numerator of this expression
+always goes to zero while the denominator never goes to zero,
+irrespectively of the widths of \(\hat{P}_{mk}\) and
+\(\hat{P}_{n}\). As such, \(\hat{S}\) always goes to zero.
+%
+\par If we use pre-convolution to avoid the AL deconvolution case, it
+is guaranteed that the matching kernel solution will be narrower in
+image space than the PSF of the science image. In frequency space, the
+relation is the opposite, and the denominator of \Cref{eq:SPre} should
+actually diverge, making the overall zero convergence faster.
+%
+\par Note that the zero convergence property of \(\hat{S}\) is kept
+even if the convergence properties are numerically lost in the second
+term of the denominator at high frequencies. Therefore we do not
+expect \emph{worse} high frequency noise to appear in our ``afterburned''
+likelihood/score image result than it would be in a simple frequency
+space PSF convolution operation.
+%
+\par We emphasize that while we can apply this correction on the
+already calculated pre-convolved AL likelihood difference image, we
+cannot do it on its variance plane. Because the noise in the score
+image is not white, we cannot estimate the noise level even in the
+homoscedastic noise model case. Instead, we need to follow the
+considerations of \Cref{sec:varplane} for the variance plane
+calculation and express \(\hat{S}\) with the overall coefficients of
+\(\hat{N}\) and \(\hat{R}\) as shown in \Cref{eq:Scncr}. Recall, we
+transform \(\hat{c}_{sn}\) and \(\hat{c}_{sr}\) back to image space,
+square them in image space, and convolve the original images' variance
+planes with these squared matching kernels. In the homoscedastic
+approximation, we multiply \(\sigma_n^2\) and \(\sigma_r^2\) with the
+sum of squares, but we still need to calculate \(c_{sn}\) and
```

Comment:
Could you add an equation or two to accompany this paragraph on the variance plane?